virtcontainers/store/vc.go

```diff
@@ -256,6 +264,11 @@ func SandboxConfigurationItemPath(id string, item Item) (string, error) {
 	return filepath.Join(ConfigStoragePath, id, itemFile), nil
 }
 
+// VCStoreUUIDPath returns a virtcontainers runtime uuid URL.
+func VCStoreUUIDPath() string {
+	return filesystemScheme + "://" + filepath.Join(VMUUIDStoragePath)
```

Comment:
`filepath.Join` with only one argument? this looks odd :thinking:

Reply:
Just used string concatenation "+" instead of join.